src/Native/gc/gc.cpp

```diff
@@ -2898,7 +2898,7 @@ void gc_heap::fire_per_heap_hist_event (gc_history_per_heap* current_gc_data_per
     current_gc_data_per_heap->gen_to_condemn_reasons.print (heap_num);
 }
 
-void gc_heap::fire_pevents()
+void gc_heap::fire_events()
```

Comment:
@Maoni0 Is the `pevents` name intentional?

Reply:
yes, it's private events.

Reply:
Got it, fixed